pkgs/development/python-modules/orange-widget-base/default.nix

```
@@ -17,12 +18,14 @@
 
 buildPythonPackage rec {
   pname = "orange-widget-base";
-  version = "4.24.0";
+  version = "4.25.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```

You will probably have to add `build-system = [ setuptools ];` too.

---

Nice catch,  I forgot that

---

Done